Snowflake.Data/Snowflake.Data.csproj

```diff
@@ -21,7 +20,7 @@
   <ItemGroup>
     <PackageReference Include="Apache.Arrow" Version="14.0.2" />
     <PackageReference Include="AWSSDK.S3" Version="3.7.0.4" />
-    <PackageReference Include="Google.Cloud.Storage.V1" Version="4.6.0" />
+    <PackageReference Include="Google.Cloud.Storage.V1" Version="3.7.0" />

```

Comment:
I don't like downgrading this dependency. I think we should resign from .netstandard.

Reply:
It's possible to keep the original package version but there would be a build warning when its set higher than `3.7.0`

Reply:
Set to the latest 4.10.0 which supports netstandard2.0